### DISCUSSION TITLE
Will ethers v6 be supported in the future?

### DISCUSSION THREAD

**Discussion by br0wnD3v** [WalletConnect/walletconnect-monorepo#2283] [Developer Support]
Hi everyone, I am not sure if web3modal does work with ethers v6 or not, as of now when getting the packages with `yarn add @web3modal/ethereum @web3modal/react wagmi ethers@^5` it runs with ethers v5. Trying to integrate an existing application running with ethers v6 and therefore is clashing with web3modal and would love to understand if its already out there or will be soon. Thank you : )

**Comment by finessevanes** [comment]
https://github.com/orgs/WalletConnect/discussions/2316
